src/networking.c

```diff
@@ -3382,6 +3382,10 @@ sds catClientInfoString(sds s, client *client, int hide_user_data) {
     }
     *p = '\0';
 
+    p = capa;
+    if (client->capa & CLIENT_CAPA_REDIRECT) *p++ = 'r';
```

Comment:
I think it would be useful to have a map like this next to the CAPA constants to do this automatically rather than adding each CAPA manually:
```
apa_map = {
    {CLIENT_CAPA_REDIRECT, 'R'},
    {CLIENT_CAPA_SSL, 'S'},
};
```

Reply:
What is the explicit value, this seems somewhat OK to me.